### DISCUSSION TITLE
does this work on ios 13?

### DISCUSSION THREAD

**Discussion by jaydenss** [dayanch96/BHTikTok-Plus#32] [General]
currently been trying to get it to work on my jailbroken ios 13 for about 30 minutes but im guessing its not supported? id PAY for this to be supported, the features are amazing! i have this installed on my iphone 13!

**Comment by dayanch96** [comment]
its for ios 14 and higher. what exactly you need to work on ios 13?

**Reply by jaydenss** [reply]
Well atm it doesn’t show up at all, no settings as if it doesn’t load

**Reply by dayanch96** [reply]
its not even installed if you’re on ios 13

**Reply by jaydenss** [reply]
Seems it, installed via sileo/zebra and also just the IPA it’s self via filza, using filza it instantly crashes when opening, then using sileo/zebra it simply doesn’t load, there is no option for the settings.

**Reply by dayanch96** [reply]
so what exactly you need? maybe ill create ios13 version

**Reply by jaydenss** [reply]
Honesty the download videos in HD option is game changing, makes stuff very clear etc and is a very cool thing no other tweak has, that’s honestly pretty much it

**Reply by dayanch96** [reply]
Just downloading regular and hd?

**Reply by jaydenss** [reply]
Yeah If possible that would be dope!
And also maybe the download photos without watermark etc, obviously having all the features would be cool but would be way to much effort. So the main things would definitely be download in HD without watermark, download photos etc :)

**Reply by anjue39** [reply]
我点下载视频，下载为空，无效

**Reply by jaydenss** [reply]
just wondering if this is still possible to do? :)

**Reply by dayanch96** [reply]
yes but unfortunately I have no plans for this
